src/tests/dbus-tests/test_10_basic.py

```diff
@@ -45,6 +45,95 @@ def test_30_supported_filesystems(self):
         self.assertEqual({str(s) for s in fss.value},
                          {'nilfs2', 'btrfs', 'swap', 'ext3', 'udf', 'xfs', 'minix', 'ext2', 'ext4', 'f2fs', 'reiserfs', 'ntfs', 'vfat', 'exfat'})
 
+    def test_40_can_format(self):
+        '''Test for installed filesystem creation utility with CanFormat'''
+        manager = self.get_interface(self.manager_obj, '.Manager')
+        with self.assertRaises(dbus.exceptions.DBusException):
+            manager.CanFormat('wxyz')
+        avail, util = manager.CanFormat('xfs')
+        if avail:
+            self.assertEqual(util, '')
+        else:
+            self.assertEqual(util, 'mkfs.xfs')
+        avail, util = manager.CanFormat('f2fs')
+        if avail:
+            self.assertEqual(util, '')
+        else:
+            self.assertEqual(util, 'mkfs.f2fs')
+        avail, util = manager.CanFormat('ext4')
+        if avail:
+            self.assertEqual(util, '')
+        else:
+            self.assertEqual(util, 'mkfs.ext4')
+        for fs in map(str, self.get_property(self.manager_obj, '.Manager', 'SupportedFilesystems').value):
+            avail, util = manager.CanFormat(fs)
+            # currently UDisks relies on executables for filesystem creation
+            if avail:
+                self.assertEqual(util, '')
+            else:
+                self.assertGreater(len(util), 0)
+
+    def test_40_can_resize(self):
+        '''Test for installed filesystem resize utility with CanResize'''
+        manager = self.get_interface(self.manager_obj, '.Manager')
+        with self.assertRaises(dbus.exceptions.DBusException):
+            manager.CanResize('nilfs2')
+        avail, mode, util = manager.CanResize('xfs')
+        # the resize mode flage values are defined in the method documentation
+        self.assertEqual(mode, 20)
```

Comment:
I think this would make more sense if it used the bitwise OR of the flags/numbers.